javascript/utils.js

```diff
@@ -77,10 +77,15 @@ var calcfilesize = function(bytes){
   return  bytes + " " + unit[index];
 };
 
+var supportImage = function(htmlToParse) {
+  var regexImg = /(img.*src=")\/([\w\/.\-\?=:;&]+")/g;
```

Comment:
make this regex non greedy `img.*?src` otherwise it'll break things if there are two image tags in an announcement